ras_backstage/Resources/secure_messaging/save_draft.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from flask import jsonify, make_response, request
+from flask_restplus import fields, Resource
+from structlog import wrap_logger
+
+from ras_backstage import secure_messaging_api
+from ras_backstage.controllers import secure_messaging_controller
+from ras_backstage.decorators.jwt_decorators import get_jwt
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@secure_messaging_api.route('/save-draft')
+class SaveDraft(Resource):
+    method_decorators = [get_jwt(request)]
+
+    draft_details = secure_messaging_api.model('DraftDetails', {
+        'msg_from': fields.String(required=True),
+        'subject': fields.String(required=True),
+        'body': fields.String(required=True),
+        'thread_id': fields.String(),
+    })
+
+    @staticmethod
+    @secure_messaging_api.expect(draft_details, validate=True)
+    @secure_messaging_api.header('Authorization', 'JWT to pass to secure messaging service', required=True)
+    def post(encoded_jwt):
+        message_json = request.get_json(force=True)
+        logger.info('Attempting to save draft')
```

Comment:
Do you need the attempting surely it can just be saving draft?

Reply:
yeah I agree I'll update throughout